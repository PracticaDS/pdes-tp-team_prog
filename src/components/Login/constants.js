export const constants = {
  title: 'Welcome to Factory Revolution',
  input: 'User',
  button: 'Login',
  placeholder: 'Login',
}
