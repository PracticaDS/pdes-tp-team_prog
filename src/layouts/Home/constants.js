export const constants = {
  title: name => `Hi ${name}! These are your factories`,
  create: 'New game',
  name: 'Name',
  actions: 'Actions',
  updated_date: 'Updated Date',
  current_currency: 'Currency',
}
