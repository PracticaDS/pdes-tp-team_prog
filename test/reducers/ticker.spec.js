import reducer from 'reducers/ticker';
import { ActionTypes } from 'constants/index';

describe('Ticker', () => {
  beforeEach(() => {});

  it(`should handle ${ActionTypes.NEXT_TICK}`, () => {
    expect(
      reducer.tick(undefined, {
        type: ActionTypes.NEXT_TICK,
        payload: {},
      }),
    ).toMatchSnapshot();
  });

  it('The store has changed', () => {
    const initialTick = { tick: 0 };
    const currentStore = reducer.tick(initialTick, { type: ActionTypes.NEXT_TICK, payload: {} });
    expect(currentStore).toEqual({ tick: 1 });
  });
});
