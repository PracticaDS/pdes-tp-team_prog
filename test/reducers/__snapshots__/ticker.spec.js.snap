// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticker should handle NEXT_TICK 1`] = `
Object {
  "tick": 1,
}
`;
